packages/experimental/RadioGroup/src/Radio/useRadio.ts

```diff
@@ -30,6 +30,8 @@ export const useRadio = (props: RadioProps): RadioInfo => {
     ...rest
   } = props;
 
+  const mergeDisabled = radioGroupContext.disabled || disabled;
```

Comment:
I've seen this called `isDisabled` elsewhere (useButton)

Reply:
Renamed! Thanks